demos/interactive_face_detection_demo/cpp/main.cpp

```diff
@@ -2,225 +2,317 @@
+DEFINE_double(t, 0.5, t_msg);
+
+constexpr char u_msg[] = "resource utilization graphs. Default is cdm. "
+    "c - average CPU load, d - load distribution over cores, m - memory usage, h - hide";
+DEFINE_string(u, "cdm", u_msg);
+
+void parse(int argc, char *argv[]) {
+    gflags::ParseCommandLineFlags(&argc, &argv, false);
+    if (FLAGS_h || 1 == argc) {
+        std::cout <<   "\t[ -h]                                         " << h_msg
+                  << "\n\t[--help]                                      print help on all arguments"
+                  << "\n\t  -m <MODEL FILE>                             " << m_msg
+                  << "\n\t[ -i <INPUT>]                                 " << i_msg
+                  << "\n\t[--bb_enlarge_coef <NUMBER>]                  " << bb_enlarge_coef_msg
+                  << "\n\t[ -d <DEVICE>]                                " << d_msg
+                  << "\n\t[--dx_coef <NUMBER>]                          " << dx_coef_msg
+                  << "\n\t[--dy_coef <NUMBER>]                          " << dy_coef_msg
+                  << "\n\t[--fps <NUMBER>]                              " << fps_msg
+                  << "\n\t[--lim <NUMBER>]                              " << lim_msg
+                  << "\n\t[--loop]                                      " << loop_msg
+                  << "\n\t[--mag <MODEL FILE>]                          " << mag_msg
+                  << "\n\t[--mam <MODEL FILE>]                          " << mam_msg
+                  << "\n\t[--mem <MODEL FILE>]                          " << mem_msg
+                  << "\n\t[--mhp <MODEL FILE>]                          " << mhp_msg
+                  << "\n\t[--mlm <MODEL FILE>]                          " << mlm_msg
+                  << "\n\t[ -o <OUTPUT>]                                " << o_msg
+                  << "\n\t[ -r]                                         " << r_msg
+                  << "\n\t[--show] ([--noshow])                         " << show_msg
+                  << "\n\t[--show_emotion_bar] ([--noshow_emotion_bar]) " << show_emotion_bar_msg
+                  << "\n\t[--smooth] ([--nosmooth])                     " << smooth_msg
+                  << "\n\t[ -t <NUMBER>]                                " << t_msg
+                  << "\n\t[ -u <DEVICE>]                                " << u_msg
+                  << "\n\tKey bindings:"
+                     "\n\t\tQ, q, Esc - Quit"
+                     "\n\t\tP, p, 0, spacebar - Pause"
+                     "\n\t\tC - average CPU load, D - load distribution over cores, M - memory usage, H - hide\n";
         showAvailableDevices();
-        return false;
-    }
-
-    if (FLAGS_i.empty()) {
-        throw std::logic_error("Parameter -i is not set");
-    }
-
-    if (FLAGS_m.empty()) {
-        throw std::logic_error("Parameter -m is not set");
+        slog::info << ov::get_openvino_version() << slog::endl;
```

Comment:
```suggestion
        std::cout << ov::get_openvino_version() << std::endl;
```
In help message it's better to avoid `[INFO]` prefix